packages/core/src/contrib/Functions.ts

```diff
@@ -669,12 +683,12 @@ export const compDict = {
    * Set the opacity of a color `color` to `frac`.
    */
   setOpacity: (color: Color<VarAD>, frac: VarAD): IColorV<VarAD> => {
-    const rgb = color.contents;
+    const props = color.contents;
     return {
       tag: "ColorV",
-      contents: {
-        tag: "RGBA",
-        contents: [rgb[0], rgb[1], rgb[2], mul(frac, rgb[3])],
```

Comment:
Since we don't have typechecking for computations, there are two near-term choices here: do nothing if the color is `INone`, or throw a runtime error.

Reply:
Added branch to do nothing in the no paint situation.

Reply:
Added a branch in this routine in the updated PR to do nothing if provided the no-paint data structure.